README.md

```diff
@@ -1,73 +1,72 @@
-# Role Name
+# VPN System Role
 
-A template for an ansible role which configures some GNU/Linux subsystem or
-service. A brief description of the role goes here.
-
-## Requirements
-
-Any pre-requisites that may not be covered by Ansible itself or the role should
-be mentioned here. For instance, if the role uses the EC2 module, it may be a
-good idea to mention in this section that the `boto` package is required.
+A Role for managing setup and configuration of VPN tunnels.
 
 ## Role Variables
 
-A description of all input variables (i.e. variables that are defined in
-`defaults/main.yml`) for the role should go here as these form an API of the
-role.
-
-Variables that are not intended as input, like variables defined in
-`vars/main.yml`, variables that are read from other roles and/or the global
-scope (ie. hostvars, group vars, etc.) can be also mentioned here but keep in
-mind that as these are probably not part of the role API they may change during
-the lifetime.
-
-Example of setting the variables:
-
-```yaml
-template_foo: "oof"
-template_bar: "baz"
-```
-
-### Variables Exported by the Role
-
-This section is optional.  Some roles may export variables for playbooks to
-use later.  These are analogous to "return values" in Ansible modules.  For
-example, if a role performs some action that will require a system reboot, but
-the user wants to defer the reboot, the role might set a variable like
-`template_reboot_needed: true` that the playbook can use to reboot at a more
-convenient time.
-
-Example:
-
-`template_reboot_needed` - default `false` - if `true`, this means
-a reboot is needed to apply the changes made by the role
-
-## Dependencies
-
-A list of other roles hosted on Galaxy should go here, plus any details in
-regards to parameters that may need to be set for other roles, or variables
-that are used from other roles.
-
-## Example Playbook
-
-Including an example of how to use your role (for instance, with variables
-passed in as parameters) is always nice for users too:
-
-```yaml
-- hosts: all
-  vars:
-    template_foo: "foo foo!"
-    template_bar: "progress bar"
-
-  roles:
-    - linux-system-roles.template
-```
+### VPN global variables (internal)
+
+System role’s global internal variables, which affect the whole system role. No platform specific variables have been defined yet.
+
+| Parameter               | Description                                         | Type |
+|-------------------------|-----------------------------------------------------|:----:|
+| vpn\_packages           | A list of packages required to set up a VPN tunnel. | list |
```

Comment:
you don't need to document variables internal to the role
internal variables should have a `__` prefix (two underscores) to denote that they are private to the role and not for use by users - see https://github.com/oasis-roles/meta_standards#naming-things